addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -28,10 +50,10 @@ options.registry.WebsiteEvent.include({
     /**
      * @override
      */
-    async _computeWidgetState(methodName, params) {
+    _computeWidgetState(methodName, params) {
```

Comment:
Not sure this change matters. The original method is async anyway, would just keep it as-is